app/elements/coder/profile/profile-page.html

```diff
@@ -127,6 +128,9 @@ <h3 class="card-header">Teams</h3>
 			},
 			_refreshTeams: function(){
 				this.$$('#teamsGrid').refresh();
+			},
+			acceptTeamInvitation: function(invitedby, teamname) {
```

Comment:
Please use caml casing here as well to be consistent with Platform and `routing.html`.